announcements/announcements.go

```diff
@@ -115,11 +115,14 @@ func remove(im *imap.Dialer, uids []int) error {
 	return err
 }
 
+var replacer = strings.NewReplacer("\n\n", "\n")
+
 func formatContents(text string) string {
+	text = replacer.Replace(text)
```

Comment:
If we are fixing the format of the message, also add a check to make sure it stays under 2000 characters.